extensions/amp-3d-gltf/validator-amp-3d-gltf.protoascii

```diff
@@ -0,0 +1,67 @@
+#
+# Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-3d-gltf
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-3d-gltf"
+    allowed_versions: "0.1"
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # <amp-3d-gltf>
+  html_format: AMP
+  tag_name: "AMP-3D-GLTF"
+  requires_extension: "amp-3d-gltf"
+  attr_lists: "extended-amp-global"
+  amp_layout: {
+    supported_layouts: FILL
+    supported_layouts: FIXED
+    supported_layouts: FIXED_HEIGHT
+    supported_layouts: FLEX_ITEM
+    supported_layouts: RESPONSIVE
+  }
+  attrs: {
+    name: "src"
+    mandatory: true
+    value_url: {
+      allow_relative: true
+      allowed_protocol: "https"
+      allowed_protocol: "http"
```

Comment:
only allow HTTPS please.

Reply:
@aghassemi if only allowing `https` then shouldn't `allow_relative` be `false`?